### ISSUE TITLE
GlowingOverscrollIndicator dartpad samples are confusing

### ISSUE THREAD

**Issue by nt4f04uNd** [flutter/flutter#76489]
GlowingOverscrollIndicator code samples may be confusing when ran in web, because glow is not drawn

https://api.flutter.dev/flutter/widgets/GlowingOverscrollIndicator-class.html

TagertPlatform should be used in these samples

**Comment by pedromassangocode** [comment]
Hi @nt4f04uNd 
Can you please elaborate on this? Do you have any visual actual vs expected results comparison?
Thank you

**Comment by HansMuller** [comment]
I agree that these examples are confusing. In part because there's only a passing mention of the fact that the GlowingOverscrollIndicator is created implicitly by ScrollBehavior.buildViewportChrome, and only on some platforms.

CC @Piinks

**Comment by Piinks** [comment]
Note for whomever executes this: The sample should use the `freeform` template so you can set the platform to android in MaterialApp.theme, otherwise the glow may not happen in the sample. (I am on a mac and so the scrollable bounces instead by default) 

Also, the current template is adding an app bar which the sample is also doing here, so there are app bars that need to be de-duped in the sample code.
